content/kb/troubleshooting-volume-readonly-or-io-error.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Troubleshooting: `volume readonly or I/O error`"
+author: Shuo Wu
+draft: false
+date: 2021-1-06
+catelogies:
+  - "HA"
+---
+
+## Applicable versions
+All Longhorn versions.
+
+## Symptoms
+When an application writes data to existing files or creates files in the mount point of a Longhorn volume, the following message is shown:
+```
+/ # cd data
+/data # echo test > test
+sh: can't create test: I/O error
+```
+
+When running the `dmesg | grep EXT4-fs` in the related pod or the node host, the message like the following is shown:
```

Comment:
I don't think we should limit it to `EXT4` here since the user might use a different filesystem for the volume.